src/components/error-message/default/index.njk

```diff
@@ -8,16 +8,16 @@ layout: layout-example.njk
 {{ govukDateInput({
   fieldset: {
     legend: {
-      text: "What is your date of birth?",
+      text: "When was your passport issued?",
       isPageHeading: true,
       classes: "govuk-fieldset__legend--xl"
     }
   },
   hint: {
-    text: "For example, 31 3 1980"
+    text: "For example, 01 08 2007"
   },
   errorMessage: {
-    text: "Date of birth must be in the past"
+    text: "Passport issue date must be in the past"
   },
   id: "dob",
```

Comment:
there's a few places where we have got 'dob' and other places where we've updated from 'dob'